### DISCUSSION TITLE
Delete/Add last Row of a table.

### DISCUSSION THREAD

**Discussion by pasteluengas** [flet-dev/flet#1021] [Q&A] [answered]
Hi, i've been trying to do this, but i cant

I tried to save the table on a variable, and use `._get_children()` to get more information and try to solve my problem.

But but it gives me an error `TypeError: Page.add() got an unexpected keyword argument 'DrumTable'`

Ideas to Delete/Add last Row of a table?

Ty

**Comment by iron3oxide** [comment, marked as answer]
~Please provide us with enough code to understand what's going on, your description alone doesn't suffice unfortunately. The part where you add the table to the page would be especially interesting.~

EDIT: just found your drummingthon repo, I guess that's what this is for. From what I can tell, there's two separate problems here: 

1. the error you get, which is very likely caused by you adding something like `page.add(DrumTable=drum_table)` to your code. Remove the `DrumTable=` part and you should be fine.
2. the question how to remove the last row of a datatable. For that, you can simply do `drum_table.rows.pop()` and then update the page/control.

**Reply by pasteluengas** [reply]
oh, Thank you. I thought the Drummingthon repository was private. But... well, it was worth leaving it public by mistake, haha

**Reply by iron3oxide** [reply]
No problem. Just updated my answer because I forgot to add `rows` before the pop() statement
